cmd/site-replication.go

```diff
@@ -2268,9 +2268,20 @@ func (c *SiteReplicationSys) toErrorFromErrMap(errMap map[string]error, actionNa
 }
 
 func (c *SiteReplicationSys) newConcErr(errMap map[string]error, actionName string) error {
+	// Sort keys of errMap to arrange errors in definitive order always
+	keys := []string{}
+	orderedErrMap := make(map[string]error, len(keys))
+	for d := range errMap {
+		keys = append(keys, d)
+	}
+	sort.Strings(keys)
+	for _, key := range keys {
+		orderedErrMap[key] = errMap[key]
```

Comment:
That is not how maps work ;)

Order is always random.

Reply:
yes, that was a mistake. Changed the logic now.